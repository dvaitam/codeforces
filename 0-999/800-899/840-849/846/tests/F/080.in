7
5 1 8 3 8 4 2
