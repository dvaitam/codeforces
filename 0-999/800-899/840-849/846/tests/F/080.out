2.918367346939
